### PR TITLE
Loan report additional values

### PR DESCRIPTION
**Pull request by SaenkoM**
Changes selector from pairs to symbols
Adds `cumulative` and `percentage` to chart

### CODE REVIEW

File: src/state/loanReport/actions.js [hunk 1]
```diff
@@ -52,35 +52,35 @@ export function updateLoanReport(payload) {
 }
 
 /**
- * Create an action to set current pair.
- * @param {string[]} pairs pairs
+ * Create an action to set current symbols.
+ * @param {string[]} symbols symbols
```
Comment (eugeneglova) [inline src/state/loanReport/actions.js]: I think it's enough `symbols` one time so no need to describe it with just name again

File: src/state/loanReport/actions.js [hunk 2]
```diff
@@ -52,35 +52,35 @@ export function updateLoanReport(payload) {
...
+    type: types.SET_SYMBOLS,
+    payload: symbols,
   }
 }
 
 /**
- * Create an action to add target pair.
- * @param {string} pair pair
+ * Create an action to add target symbol.
+ * @param {string} symbol symbol
```
Comment (eugeneglova) [inline src/state/loanReport/actions.js]: same here, we can remove `symbol` as description in JSDOC as it's the same as name of param

File: src/state/loanReport/actions.js [hunk 3]
```diff
@@ -52,35 +52,35 @@ export function updateLoanReport(payload) {
...
+    type: types.ADD_SYMBOL,
+    payload: symbol,
   }
 }
 
 /**
- * Create an action to remove target pair.
- * @param {string} pair pair
+ * Create an action to remove target symbol.
+ * @param {string} symbol symbol
```
Comment (eugeneglova) [inline src/state/loanReport/actions.js]: here too

File: src/state/loanReport/actions.js (hunk 1 above)
Comment (SaenkoM) [reply src/state/loanReport/actions.js]: going to fix that in a separate PR since there are many such cases

**Pull request merged**